std_msgs/Float64 heading
std_msgs/Float64 distance
geometry_msgs/Point start
geometry_msgs/Point finish
std_msgs/Int64 numPoints
std_msgs/Float32 wpSpace